### PR TITLE
Content update on the page "Charmed Kubernetes features"

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Added a link to https://ubuntu.com/openstack on the website: https://ubuntu.com/kubernetes/features
- Changed "Logging and monitoring" section's title to "Observability and metrics"

## QA

- Open the website: https://ubuntu-com-10361.demos.haus
- Make sure that there is a title called "Observability and metrics" 
- Make sure that OpenStack is a link in the section: https://ubuntu-com-10361.demos.haus/kubernetes/features#bare-metal

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4431

### CODE REVIEW

**Comment by bethcollins92** [comment]
Hey @goulinkh - looking at the copy doc there is an extra "Shape it to your needs" section which needs to be added and the order of content is slightly different.

**Comment by goulinkh** [comment]
> looking at the copy doc there is an extra "Shape it to your needs" section which needs to be added

I wasn't sure whether to add this part or not, I will look into it

File: templates/kubernetes/features.html [hunk 1]
```diff
@@ -68,44 +68,40 @@ <h3 class="p-card__title">Managed Kubernetes</h3>
...
-      <h2>Charmed Kubernetes fits perfectly on top of OpenStack, VMware, and bare metal</h2>
-      <p>There's no doubt that Kubernetes is the new standard operational layer for every multi-cloud business. Canonical enables K8s on demand for your DevOps teams &mdash; on OpenStack, VMware, public clouds, and bare metal clusters with MAAS.</p>
+    <div class="col-6">
+      <h2>Shape it to your needs</h2>
     </div>
   </div>
   <div class="row">
-    <div class="col-8">
-      <p>With Charmed Kubernetes, you can deliver 'Containers as a Service' across the enterprise, to enable each project to spin up a standardised K8s environment of arbitrary scale, on demand, with centralised operational control.</p>
+    <div class="row u-equal-heigh">
```
Comment (bethcollins92) [inline templates/kubernetes/features.html:79]: There shouldn't be a row nested in a row (you only need one). Also - spelling mistake:
```suggestion
    <div class="row u-equal-height">
```

File: templates/kubernetes/features.html [hunk 2]
```diff
@@ -68,44 +68,40 @@ <h3 class="p-card__title">Managed Kubernetes</h3>
...
-      <li class="p-list__item is-ticked">Prometheus and ELK logging and monitoring included</li>
-      <li class="p-list__item is-ticked">Extensible, third-party ecosystem for storage and networking</li>
-    </ul>
-  </div>
-
-  <div class="row">
-    <div class="col-8">
-      <p>In partnership with Google, Canonical delivers the perfect K8s for AI and machine learning with Kubeflow.</p>
-      <p><a href="/ai/what-is-kubeflow">Watch how to build a machine learning model on Ubuntu and Kubeflow&nbsp;&rsaquo;</a></p>
+    <div class="row u-equal-heigh">
```
Comment (bethcollins92) [inline templates/kubernetes/features.html:93]:
```suggestion
    <div class="row u-equal-height">
```

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good! Couple of comments in the code, and a comment in the copy doc about a different title. I've also requested different text for the "Learn more" links so if Alex or Tytus gets back to me could you update those? Thanks

File: templates/kubernetes/features.html (hunk 1 above)
Comment (goulinkh) [reply templates/kubernetes/features.html:79]: Oh my bad sorry for the typo, I will look into the nested row, thanks :)

**Comment by goulinkh** [comment]
@bethcollins92 For some reason the demo link hasn't been updated

**Review by bethcollins92 — APPROVED** [review]
LGTM 🎉  - thanks for making the update! @alxchalkias could you check it please? (FYI the contact form PR is still being worked on)

**Comment by alxchalkias** [comment]
@bethcollins92 can I have the link to the latest demo please? The ones I see in the comments still include things that shouldn't go live

**Comment by bethcollins92** [comment]
@alxchalkias Demo should be updated now https://ubuntu-com-10361.demos.haus/kubernetes/features

**Comment by alxchalkias** [comment]
+1 from me too! Thank you

**Pull request merged**